integreat_compass/cms/templates/interactions/vote_form.html

```diff
@@ -0,0 +1,67 @@
+{% extends "_base.html" %}
+{% load i18n %}
+{% load static %}
+{% load widget_tweaks %}
+{% load render_bundle from webpack_loader %}
+{% block title %}
+    "todo"
+{% endblock title %}
+{% block content %}
+    <div>
+        {% include "includes/dashboard_navbar.html" %}
+        {% for offer_version in pending_offer_versions %}
+            <form action="{% url 'cms:protected:add_vote' offer_version.id %}"
+                  method="post">
+                {% csrf_token %}
+                <div class="flex flex-row border-2 border-gray-200 rounded-lg shadow-md shadow-inner p-6 mb-8">
+                    <div class="basis-2/3 pr-8">
```

Comment:
On mobile and tablet sizes, the content overflows the card a little bit. But since in one of the meetings we said, that this is not highly important for the MVP, I guess it's okay?

Reply:
fixed it.